src/User-repo.js

```diff
@@ -1,23 +1,27 @@
-import User from "./User";
-
 class UserRepo {
   constructor(users, currentUser) {
     this.users = users;
     this.currentUser = currentUser;
-    this.averageAllStepGoals = this.users.map(user => user.stepGoal)/this.users.length;
+    this.averageAllStepGoals = this.calculateAverageStepGoal();
+    // this.averageAllStepGoals = this.users.reduce((stepSum, user) => {
+    //   stepSum += user.dailyStepGoal;
+    //   return stepSum;
+    // }, 0)/this.users.length;
   }
-  getDataFromID() {
-    return this.users.find((user) => this.currentUser.id === user.id);
+  getDataFromID(id) {
```

Comment:
Does passing in id make it dynamic for other users? Where do we invoke this method?